### ISSUE TITLE
[Suggestion] Calibration mode for accel mouse angle

### ISSUE THREAD

**Issue by Saqib117** [billycao/Wi-Mote#3]
add a means for the user to calibrate the "level" of the accel mouse mode, so that they don't have to keep the phone completely level. They may find it more comfortable at a small angle. 

**(i noticed this more during gaming mode when i naturally tilted my hand back, and the mouse would move up)

**Comment by Sayan-** [comment]
We could add a button similar to a TARE on a scale where we use this new angle (or whatever the raw sensor data is) as the baseline for future calculations. Every time they hit our version of TARE it would re. Add a reset and it could be the easy fix.

**Comment by Saqib117** [comment]
Yeah, that's already something I was planning on adding, I just wanted a place to dump my plans and share them for now. i'm gonna make a lot of accel changes and improvements spring break and next quarter.
